packages/kit/src/exports/index.js

```diff
@@ -53,3 +53,5 @@ export function fail(status, data) {
 export function invalid() {
 	throw new Error('invalid(...) is now fail(...)');
 }
+
+export { HttpError, Redirect };
```

Comment:
oh wait, this is just left over from a previous exploration. it can be removed i think (though will double-check locally before doing so)

```suggestion
```